src/ImageSharp/Formats/Png/PngChunk.cs

```diff
@@ -36,7 +36,16 @@ public PngChunk(int length, PngChunkType type, IMemoryOwner<byte> data = null)
     /// Gets the data bytes appropriate to the chunk type, if any.
     /// This field can be of zero length or null.
     /// </summary>
-    public IMemoryOwner<byte> Data { get; }
+    public IMemoryOwner<byte>? Data { get; }
+
+    public ReadOnlySpan<byte> DataSpan
```

Comment:
Do you have a better idea to to this? Or should I check for null at all consuming places?